### PR TITLE
Update tooltip docs

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

**Comment by domoritz** [comment]
Waiting for a Vega fix.

**Comment by domoritz** [comment]
Ready @kanitw

File: site/docs/tooltip.md [hunk 1]
```diff
@@ -5,26 +5,30 @@ title: Tooltip
...
 
-To quickly create a tooltip without a plugin, Vega-lite's [`tooltip`]({{site.baseurl}}/docs/encoding.html#mark-properties-channels) channel can be mapped to a data field. For example, this bar chart supports tooltips for field `b`. Hover over the bar and notice the simple tooltip that displays the value of field `b` for each bar.
+To create a tooltip, Vega-Lite's [`tooltip`]({{site.baseurl}}/docs/encoding.html#mark-properties-channels) channel can be mapped to a data field. For example, this bar chart supports tooltips for field `b`. Hover over the bar and notice the simple tooltip that displays the value of field `b` for each bar.
 
 <div class="vl-example" data-name="bar_tooltip"></div>
 
-To show more than one field, you can calculate a new field that concatenates multiple fields. You can further customize the tooltip by specifying a custom event handler that gets invoked every time tooltip displays via [`tooltipHandler`](https://vega.github.io/vega/docs/api/view/#view_tooltipHandler)of the [`Vega View API`](https://vega.github.io/vega/docs/api/view/).
+To show more than one field, you can provide an array of fields or calculate a new field that concatenates multiple fields.
```
Comment (kanitw) [inline site/docs/tooltip.md]: > provide an array of fields

an array of field definitions

> or calculate a new field that concatenates multiple fields.

Not very clear from the text that this means literally using `calculate` transform?

File: site/docs/tooltip.md [hunk 2]
```diff
@@ -5,26 +5,30 @@ title: Tooltip
...
+You can further customize the tooltip by specifying a custom event handler via [`tooltipHandler`](https://vega.github.io/vega/docs/api/view/#view_tooltipHandler) of the [`Vega View API`](https://vega.github.io/vega/docs/api/view/). Vega invokes the handler every time a tooltip is shown
 
-<link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/vega-tooltip@{{ site.data.versions.vega-tooltip }}/build/vega-tooltip.css">
-<div class="vl-example tooltip" data-name="bar"></div>
+We provide [`vega-tooltip`](https://github.com/vega/vega-tooltip/), a tooltip handler that creates a customizable HTML tooltip. Below is an example of Vega-lite visualization with [`vega-tooltip`](https://github.com/vega/vega-tooltip/) plugin.
+
+<div class="vl-example tooltip" data-name="bar_tooltip"></div>
+
+Without the tooltip plugin, Vega-Lite will use native tooltips. Move your cursor over one of the bars to see it (you might have to wait for a little bit for the tooltip to appear).
```
Comment (kanitw) [inline site/docs/tooltip.md]: > use native tooltips.

"generate tooltips via native HTML `"title"` attribute." ?

File: site/docs/tooltip.md [hunk 3]
```diff
@@ -5,26 +5,30 @@ title: Tooltip
 permalink: /docs/tooltip.html
 ---
 
-Tooltip can provide details of a particular data point on demand. There are two ways to create a tooltip in Vega-Lite.
+Tooltip can provide details of a particular data point on demand. Tooltips are created with the `tooltip` channel. By default, the renderer with use native tooltips.
```
Comment (kanitw) [inline site/docs/tooltip.md]: > with

You mean "will"?

> use native tooltips.

"generate tooltips via native HTML "title" attribute." ?

File: site/docs/tooltip.md (hunk 1 above)
Comment (domoritz) [reply site/docs/tooltip.md]: Fixed

File: site/docs/tooltip.md (hunk 2 above)
Comment (domoritz) [reply site/docs/tooltip.md]: Done

File: site/docs/tooltip.md (hunk 3 above)
Comment (domoritz) [reply site/docs/tooltip.md]: Done

**Pull request merged**